package.json

```diff
@@ -24,16 +26,9 @@
       "react-app/jest"
     ]
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+  "browserslist": [
```

Comment:
Is there any specific reason to remove the development list? I feel it's better we don't touch this part of package.json because this gives a good development experience, especially when using language features such as async/await, but still provides high compatibility with many browsers in production. Let me know your thoughts.

Reply:
The reason for that is the nullish coalescing operator (??).

When this operator appears in any of the node_modules, it gives out an error. Since we are using react-leaflet package for displaying the map, which uses this operator, we were getting the following error:

![image](https://user-images.githubusercontent.com/56843437/142366678-a8dda0a6-b00e-4147-b1e9-54c867cec9da.png)


So we researched around it and found the following fix: [stackoverflow](https://stackoverflow.com/questions/67552020/how-to-fix-error-failed-to-compile-node-modules-react-leaflet-core-esm-pat)

Also, it seems to be an issue with the create-react-app. It can be looked more into from [Facebook/create-react-app issue](https://github.com/facebook/create-react-app/issues/9468).

Reply:
Oh alright! Hope it doesn't break anything but seems like this is the only way around.